cli/cmd/generate_test.go

```diff
@@ -16,7 +16,7 @@ spec:
   name: "test"
 
   # Version of the plugin to use.
-  version: "v1.1.4"
+  version: "v1.1.5"
```

Comment:
Every time we release a new test plugin we need to update this, it's fixed in https://github.com/cloudquery/cloudquery/pull/1848/files#diff-d47c553087688ec44981249565e43b874cdc5f189894bd10829dcf0626831af0R53 by check the version is a valid SemVer one instead of a string comparison